packages/amf-deserializer/CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Change Log
```

Comment:
These are from different PRs/commits , we need to rebase

Reply:
Is this automatic as part of lerna? Would be nice..

Reply:
Lerna creates changelogs automatically, but you have commits from a different context, that has nothing to do with Lerna: )

Reply:
I checked the differences between this PR and the master branch. The only other changes are to the README.md files in packages/io, and the addition of test/helpers/nearlyEqual.js

Is there something else?

Reply:
I see a lot of changes in other packages as well, I will try to do a rebase locally to see if that would fix things

Reply:
Any chance that you have a old copy of IO master? Hmm...

Reply:
I have an up to date master, so that is all we need :) this should be saved for last :)